R/methods-Collections.R

```diff
@@ -32,32 +32,69 @@ setGeneric("getCollectionNames", function(object) standardGeneric("getCollection
 #' @aliases getCollectionNames,Collections-method
 setMethod("getCollectionNames", "Collections", function(object) return(sapply(object, name)))
 
+#' Get Collection Variable Names
+#' 
+#' Get the names of the variables in the Collection
+#' @param object A Collection
+#' @param ... Additional arguments
```

Comment:
do we have additional arguments?

Reply:
some of the methods do.. the generic is shared across packages